modules/social_features/social_event/src/Plugin/views/sort/EventDate.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
@dmytriikaiun I think instead of writing new plugin, we can just change the logic of existing `event_passed_upcoming_sort` - since it will be not used.

@ronaldtebrake, @bramtenhove, or do you think we need to leave it and mark ass deprecated?

Reply:
@ribel 
I think this is actually a nice solution :) if people still use it in different views which we don't provide this will help them out and they can change it themselves. Marking the other as deprecated in favor of this new one is a good solution!

Reply:
@dmytriikaiun Good, could you then just `@deprecated` annotation to `event_passed_upcoming_sort` plugin?

Reply:
@ribel Yes, I could